src/main/java/org/gridsuite/study/server/dto/LoadFlowParametersValues.java

```diff
@@ -20,10 +18,9 @@
 @Getter
 @AllArgsConstructor
 @Builder
-@NoArgsConstructor
 public class LoadFlowParametersValues {
 
-    private LoadFlowParameters commonParameters;
+    private final LoadFlowParameters commonParameters;
 
-    private Map<String, Map<String, Object>> specificParametersPerProvider;
+    private final Map<String, String> specificParameters;
 }
```

Comment:
Just a note: you can use record now
```suggestion
@Builder
public record LoadFlowParametersValues(
    LoadFlowParameters commonParameters,
    Map<String, String> specificParameters
 ) { }
```